ApiCompatTask/source/commandLineResult.ts

```diff
@@ -0,0 +1,48 @@
+import { TaskResult, getInput } from "azure-pipelines-task-lib";
+
+export default class CommandLineResult {
+    private _totalIssues: number;
+    private _body: string;
+
+    get totalIssues() {
+        return  this._totalIssues;
+    }
+
+    get body() {
+        return this._body;
+    }
+
+    constructor( result: string ) {
+        const indexOfTotalIssues = result.indexOf("Total Issues");
+        this._totalIssues = this.getTotalIssues(result, indexOfTotalIssues);
+        this._body = this.getBody(result, indexOfTotalIssues);
+    }
+
+    private getTotalIssues = (message: string, indexOfTotalIssues: number): number => {
+        return parseInt(message.substring(indexOfTotalIssues).split(':')[1].trim(), 10);

```

Comment:
Is there any case where this `split` method returns `null` or only one item?

Reply:
Nope! the task will always return 

> Total Issues : NUMBER